tools/validators/ontology_validator/yamlformat/validator/subfield_lib.py

```diff
@@ -248,28 +247,27 @@ class Subfield(findings_lib.Findings):
       SubfieldCategory value representing the subfield type.
     description: explanation of what this subfield represents. Optional (for
       now) string semantic definition for the subfield.
-    context: the config file context for where this subfield was defined.
+    file_context: the config file context for where this subfield was defined.
       Optional object with the config file location of this subfield.
 
   Returns:
     An instance of the Subfield class.
   """
 
-  def __init__(self, name, category, description=None, context=None):
+  def __init__(self, name, category, description=None, file_context=None):
```

Comment:
args?

Reply:
Done.

Reply:
The docstring is there for state_lib.py:154